File: ios/EmulatorDetectionModule.m
//
//  EmulatorDetection.m
//  exageDemom
//
//  Created by apple on 3/20/22.
//

#import <Foundation/Foundation.h>
#import "React/RCTBridgeModule.h"

@interface
RCT_EXTERN_MODULE(EmulatorDetectionModule, NSObject)
RCT_EXTERN_METHOD(isEmulator : (RCTResponseSenderBlock)callback)

@end

File: ios/EmulatorDetectionModule.swift
//
//  EmulatorDetectionModule.swift
//  exageDemom
//
//  Created by apple on 3/21/22.
//

import Foundation

@objc(EmulatorDetectionModule)
class EmulatorDetectionModule : NSObject{
  @objc
   func isEmulator(_ callback:RCTResponseSenderBlock) -> Bool {
    var flag=false;
#if targetEnvironment(simulator)
    flag=true;
#endif
callback([NSNull(),flag])
    return flag;
  }
}

